Portfolio/src/Wrapper.vue

```diff
@@ -0,0 +1,34 @@
+<script setup lang="ts">
+</script>
+
+<template>
+  <div class="styled-wrapper">
+
+    <slot></slot>
+  </div>
+</template>
+
+<style scoped>
+.styled-wrapper {
+  font-family: "Raleway";
```

Comment:
Font-families should be declared in their own file as a good practice.
You can add global style files from where you can declare font-faces etc.

Reply:
Done!